chrome/content/zotero/xpcom/sync/syncLocal.js

```diff
@@ -154,6 +156,30 @@ Zotero.Sync.Data.Local = {
 	},
 	
 	
+	removeLegacyLogins: function () {
+		var loginManagerHost = 'chrome://zotero';
+		var loginManagerRealm = 'Zotero Sync Server';
+		
+		Zotero.debug('Removing legacy Zotero sync credentials (api key acquired)');
+		
+		var loginManager = Components.classes["@mozilla.org/login-manager;1"]
+			.getService(Components.interfaces.nsILoginManager);
+		loginManager.removeAllLogins();
```

Comment:
Yeah, don't want this — this would wipe out all the user's saved passwords in Firefox.